src/neuigkeiten/2022-04-06-release/index.md

```diff
@@ -0,0 +1,58 @@
+---
+draft: false
+layout: layout-neuigkeiten
+date: 2022-04-06
+title: Konfiguration von Sonderurlaubstagen und Bildungsurlaub
+author: Tobias Schneider
+release-version: 4.34.0
+teaserImage: /static/update.jpg
+---
+
+Sonderurlaube können individualisiert werden und Bildungsurlaub steht als neue Abwesenheitsart zur Verfügung
+
+<!-- more -->
+
+### Konfiguration der Sonderurlaubstage
+
+Mit der Version 4.34.0 zieht die neue Konfiguration **Einstellungen zu Sonderurlaub** in die Urlaubsverwaltung ein.
+Ihr findet diese als Person mit der Berechtigung _Office_ unter _Einstellungen > Abwesenheitsarten_. An dieser Stelle
+könnt ihr nun die Anzeige der bei euch im Unternehmen möglichen Sonderurlaube sowie die Anzahl der Tage konfigurieren.
```

Comment:
```suggestion
könnt ihr nun die Anzeige der bei euch in der Organisation möglichen Sonderurlaube sowie die Anzahl der Tage konfigurieren.
```

Nicht alle Kunden sind Unternehmen? Weiß aber nicht in wie weit wir das an anderer Stelle beachten. Eventuell ist dann konsistent von Firma/Unternehmen zu sprechen besser.